src/components/Input/NumericInput/NumericInput.tsx

```diff
@@ -126,6 +135,7 @@ export function NumericInput({
           />
         )
       }
+      error={isOutsideRange ? `Value outside range [${min}, ${max}]` : undefined}
```

Comment:
how about only showing the error message when editing?  IIRC that's how our last UI worked at least, but it might be worth discussing if that's what we really want. 
```suggestion
      error={isEditing && isOutsideRange ? `Value outside range [${min}, ${max}]` : undefined}
```

Reply:
I think that would be good, if I've knowingly set a value outside the range a red warning text would be quite annoying to see all the time. I could however see that we add some kind of exclamation or something in yellow/orange to indicate which ones have values outside

Reply:
Tried something lmk what you think